openfin/fin-trader
# Hyp 52 pre existing bug some blotter win

What to test in this PR
1. Click on purchase for a particular window, after the notification has shown on the window, crosscheck to see that the HIGH, OPEN and LOW prices are not set to a constant as seen in image below
2. Test the chartiq link for all windows AWH and ORI specifically
3. Check that the main price on the window is fluctuating now and the OPEN price is a constant
<img width="381" alt="screen shot 2017-06-19 at 6 21 23 pm" src="https://user-images.githubusercontent.com/10234875/27348025-94e1c252-55ea-11e7-80e2-6fdcfcac6c9b.png">
